AttoBot/lambda_function.py

```diff
@@ -25,6 +25,12 @@
 
 SECRET = os.environ["SECRET"]
 
+TAG_REQ = "\n".join((
+    "Please make sure that:",
+    "- [ ] CI passes for supported Julia versions (if applicable).",
+    "- [ ] Version bounds are up to date."
```

Comment:
That would be a huge boon as far as distributing the workload is concerned, I think. I suspect even many package authors aren't sure exactly what they should be doing here.